dune/xt/la/test/eigensolver_for_real_matrix_with_real_evs_from_3d_pointsource.tpl

```diff
@@ -195,22 +159,12 @@ TEST_F(EigenSolverForMatrixFrom3dPointsource_{{T_NAME}}, is_constructible)
 
 TEST_F(EigenSolverForMatrixFrom3dPointsource_{{T_NAME}}, gives_correct_eigenvalues)
 {
-  gives_correct_eigenvalues({ {"eigen", "1e-13"}, {"shifted_qr", "1e-14"} });
-}
-
-TEST_F(EigenSolverForMatrixFrom3dPointsource_{{T_NAME}}, gives_correct_eigenvalues_in_correct_order)
-{
-  gives_correct_eigenvalues_in_correct_order({ {"eigen", /*we_expect_a_failure: */ "-1"}, {"shifted_qr", /*we_expect_a_failure: */ "-1"} });
+  gives_correct_eigenvalues({ {"lapack", "1e-14"}, {"eigen", "1e-13"}, {"shifted_qr", "1e-14"}});
```

Comment:
You probably also need to add the same tolerances for the numpy solver, as it uses LAPACK internally.